code/__DEFINES/dcs/signals/signals_atom/signals_atom_attack.dm

```diff
@@ -57,6 +57,8 @@
 	#define COMPONENT_IMMUNE_TO_TILT_AND_CRUSH (1<<0)
 ///Called when a atom gets something tilted on them
 #define COMSIG_POST_TILT_AND_CRUSH "atom_post_tilt_and_crush"
+/// Called when an atom is splashed with something
```

Comment:
Use the regular format of including the args in the comment, I'd appreciate you adding it to the others here as well